### PR TITLE
Document auxiliary kernels from THM

### PR DESCRIPTION
**Pull request by GiudGiud**
not the most important pages again but in the background I m learning THM, hopefully.

### CODE REVIEW

**Review by joshuahansel — APPROVED** [review]

File: modules/thermal_hydraulics/doc/content/source/functions/GeneralizedCircumference.md [hunk 1]
```diff
@@ -1,20 +1,14 @@
...
 !syntax description /Functions/GeneralizedCircumference
 
-## Overview
-
-!! Replace these lines with information regarding the GeneralizedCircumference object.
+The expression for the generalized circumference is:
 
-## Example Input File Syntax
+!equation
+circumference = \sqrt{4 \pi area + (\dfrac{dA}{dx})^2}
```
[the remark this answers is not in the document]
Comment (GiudGiud) [reply modules/thermal_hydraulics/doc/content/source/functions/GeneralizedCircumference.md:8]: yeah had the same reaction.
@andrsd would you mind explaining that to us

[the remark this answers is not in the document]
Comment (andrsd) [reply modules/thermal_hydraulics/doc/content/source/functions/GeneralizedCircumference.md:8]: All I know is this:

https://github.com/idaholab/moose/blob/968eb46a3868f624d7930083ffdedd039b3d4fd5/modules/thermal_hydraulics/include/functions/GeneralizedCircumference.h#L15-L30

[the remark this answers is not in the document]
Comment (joshuahansel) [reply modules/thermal_hydraulics/doc/content/source/functions/GeneralizedCircumference.md:8]: So basically, just to obtain some length-averaged value of the perimeter 🤷‍♂️

**Comment by joshuahansel** [comment]
Oh yes, the classic 7 approving reviews needed trick :P

**Pull request merged**